js/geo/transform.js

```diff
@@ -127,6 +127,53 @@ Transform.prototype = {
         this._constrain();
     },
 
+    /**
+     * Return a zoom level that will cover all tiles the transform
+     * @param {Object} pyramidProperties
+     * @param {number} pyramidProperties.tileSize
+     * @param {boolean} pyramidProperties.roundZoom
+     * @returns {number} zoom level
+     * @private
+     */
+    coveringZoomLevel: function(pyramidProperties) {
```

Comment:
The fact that these properties were once part of `TilePyramid` isn't important going forward. Let's just use a generic name like `options` 😄